prql-compiler/src/codegen.rs

```diff
@@ -295,8 +354,8 @@ fn binding_strength(expr: &pl::ExprKind) -> i32 {
             pl::BinOp::Or => 1,
             pl::BinOp::Coalesce => 2,
         },
-        pl::ExprKind::Unary { .. } => 9,
-        pl::ExprKind::FuncCall(_) => 0,
+        pl::ExprKind::Unary { .. } => 6,
+        pl::ExprKind::FuncCall(_) => 7,
         pl::ExprKind::Func(_) => 0,
 
         _ => 11,
```

Comment:
Why do other nodes have stronger binding than idents?

Reply:
I told you not to look at the code!! :D